### ISSUE TITLE
求一輸入函數

### ISSUE THREAD

**Issue by fishfishfishfishfish** [LingDong-/wenyan-lang#137]
Is there an input statement?

**Comment by GLanguage** [comment]
Yes. We REALLY NEED an input statement/package. It is very hard (or impossible) to make a number-guessing game or something like that in wenyan without any input.
Actually, there IS an input package called 'readline' in node.js. Maybe we can write one in wenyan.

**Comment by LingDong-** [comment]
Good suggestion, input function will likely be packaged in platform-specific IO stdlib .
